app-arch/unar/unar-1.10.6.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs bash-completion-r1
+
+DESCRIPTION="unpacker for various archiving formats, e.g. rar v3"
+HOMEPAGE="https://unarchiver.c3.cx/"
+SRC_URI="
+	https://github.com/MacPaw/XADMaster/archive/v${PV}/XADMaster-${PV}.tar.gz
+	https://github.com/MacPaw/universal-detector/archive/1.1/universal-detector-1.1.tar.gz
+"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+RDEPEND="
+	media-sound/wavpack
+	app-arch/bzip2:=
+	dev-libs/icu:=
+	gnustep-base/gnustep-base
+	sys-libs/zlib:=
+"
+DEPEND="${RDEPEND}
+	app-arch/unzip
+	>=gnustep-base/gnustep-make-2.6.0[native-exceptions]
+	sys-devel/gcc[objc]
+"
+
+S="${WORKDIR}/XADMaster-${PV}"
+
+src_prepare() {
+	default
+	# avoid jobserver warning, upstream bug:
+	# https://bitbucket.org/WAHa_06x36/theunarchiver/issues/918/dont-call-make-from-makefile
+	sed -i -e 's:make:$(MAKE):g' Makefile.linux || die
+	mv "${WORKDIR}/universal-detector-1.1" "${WORKDIR}/UniversalDetector"
+}
+
+src_compile() {
+	emake -f Makefile.linux \
+		AR="$(tc-getAR)" \
+		CC="$(tc-getCC)" \
+		CXX="$(tc-getCXX)" \
+		OBJCC="$(tc-getCC)" \
+		CFLAGS="${CFLAGS}" \
+		CXXFLAGS="${CXXFLAGS}" \
+		OBJCFLAGS="${CFLAGS}" \
+		LD="$(tc-getCXX)" \
+		LDFLAGS="-Wl,--whole-archive -fexceptions -fgnu-runtime ${LDFLAGS}"
+}
+
+src_install() {
+	dobin {ls,un}ar
+	doman Extra/{ls,un}ar.1
+	dobashcomp Extra/{ls,un}ar.bash_completion
+}
```

Comment:
Are there any documention to be installed, like README* files?

Reply:
I could not find any documentation.